lib/plugin/videoroom.ex

```diff
@@ -52,8 +52,10 @@ defmodule Janus.Plugin.VideoRoom do
     with {:ok, %{"videoroom" => "created", "room" => id}} <-
            Session.execute_request(session, message) do
       {:ok, id}
+    else
+      error ->
+        Errors.handle(error)
```

Comment:
A bit of nitpicking, but can you make it one line? Applies to other occurrences of Errors.handle()